fuse_examples/imaging/segmentation/picai/conf/config_template.yaml

```diff
@@ -0,0 +1,53 @@
+paths:
+  working_dir : .
+  model_dir : ${paths.working_dir}/model_dir
+  inference_dir : ${paths.working_dir}/model_new/infer_dir
+  eval_dir : ${paths.working_dir}/model_new/eval_dir
+  cache_dir : ${paths.working_dir}/cache/PICAI
+  clinical_file : ${oc.env:PICAI_DATA_PATH}/labels/picai_labels-main/clinical_information/marksheet.csv
```

Comment:
Did you add PICAI_DATA_PATH to env.sh file on CCC?

Reply:
yes, otherwise the test would not run...